src/Codeception/Lib/Connector/Yii2/TransactionForcer.php

```diff
@@ -16,22 +15,17 @@
  */
 class TransactionForcer extends ConnectionWatcher
 {
-    private $ignoreCollidingDSN;
+    private bool $ignoreCollidingDSN;
+    private array $pdoCache = [];
+    private array $dsnCache = [];
+    private array $transactions = [];
 
-    private $pdoCache = [];
-
-    private $dsnCache;
-
-    private $transactions = [];
-
-    public function __construct(
-        $ignoreCollidingDSN
-    ) {
+    public function __construct(bool $ignoreCollidingDSN)
```

Comment:
```suggestion
    public function __construct(private readonly bool $ignoreCollidingDSN)
```

Reply:
I'd personally vote to increase the minimum version to 8.3. And we make this a major version bump.
https://www.php.net/supported-versions

Reply:
Do this in a different PR.